app-frontend/src/app/core/services/auth.service.js

```diff
@@ -218,11 +224,27 @@ export default (app) => {
         }
 
         verifyAuthCache() {
+            let token = this.token();
```

Comment:
Ugh, yeah; Swift and Rust use `let` to denote a constant so I always get that mixed up.